### PR TITLE
Task 4

### PR DESCRIPTION
**Pull request by gracu-koduje**
Please code review.

### CODE REVIEW

File: js/script.js [hunk 1]
```diff
@@ -1,8 +1,9 @@
-/*jslint unparam: true, sloppy: true*/
+/*jslint devel: true, unparam: true, sloppy: true*/
 /*jshint es3: false, browser: true*/
 
 /* -================WELCOME IN JS FUN-MODE====================-*/
 
+//Task number 3
 function Mytest() {
```
Comment (matwjablonski) [inline js/script.js]: unnecessary empty line and rename function

File: js/script.js [hunk 2]
```diff
@@ -1,8 +1,9 @@
...
 
     var a, h, triangleArea;
```
Comment (matwjablonski) [inline js/script.js]: vars in new lines

File: js/script.js (hunk 1 above)
Comment (gracu-koduje) [inline js/script.js]: Done

File: js/script.js (hunk 2 above)
Comment (gracu-koduje) [inline js/script.js]: Ok.

**Pull request merged**